ember-server/jvm/src/test/scala/org/http4s/ember/server/EmberServerMtlsSuite.scala

```diff
@@ -125,6 +126,7 @@ class EmberServerMtlsSuite extends Http4sSuite {
       tlsContext <- authTlsClientContext
       emberServer <- EmberServerBuilder
         .default[IO]
+        .withPort(ip4s.Port.fromInt(0).get)
```

Comment:
Not a blocker but would be nice to have a string interpolator for this

Reply:
Ah, that's on me, sorry. We should upstream that change from 0.23 actually.

Reply:
No worries, this wasn't really the place to bring it up anyway